docs/clusterdefinition.md

```diff
@@ -471,16 +471,18 @@ We consider `kubeletConfig`, `controllerManagerConfig`, `apiServerConfig`, and `
+| firstConsecutiveStaticIP     | only required when vnetSubnetId specified and when MasterProfile is not `VirtualMachineScaleSets`  | The IP address of the first master. IP Addresses will be assigned consecutively to additional master nodes. When MasterProfile is using `VirtualMachineScaleSets`, this value will be determined by an offset from the first IP in the `vnetCidr`. For example, if `vnetCidr` is `10.239.0.0/16`, then `firstConsecutiveStaticIP` will be `10.239.0.4`                                                                                                                                                                                                                                                                                                                 |
 | vmsize                       | yes                                       | Describes a valid [Azure VM Sizes](https://azure.microsoft.com/en-us/documentation/articles/virtual-machines-windows-sizes/). These are restricted to machines with at least 2 cores and 100GB of ephemeral disk space                                                                                                                                                                                                     |
 | osDiskSizeGB                 | no                                        | Describes the OS Disk Size in GB                                                                                                                                                                                                                                                                                                                                                                                           |
-| vnetSubnetId                 | no                                        | Specifies the Id of an alternate VNET subnet. The subnet id must specify a valid VNET ID owned by the same subscription. ([bring your own VNET examples](../examples/vnet))                                                                                                                                                                                                                                                |
+| vnetSubnetId                 | only required when using custom VNET                                        | Specifies the Id of an alternate VNET subnet. The subnet id must specify a valid VNET ID owned by the same subscription. ([bring your own VNET examples](../examples/vnet)). When MasterProfile is using `VirtualMachineScaleSets`, this value should be the subnetId of the master subnet.                                                                                                                                                                                                                                                |
```

Comment:
`When MasterProfile is using `VirtualMachineScaleSets`, this value should be the subnetId of the master subnet` --> what does that imply for the case when master profile is using VMAS?